examples/monaco-react/pages/index.tsx

```diff
@@ -0,0 +1,21 @@
+'use client'
+
+import React, { useEffect, useRef } from 'react';
```

Comment:
This should be `app/page.tsx` and you'll also need a root layout `app/layout.tsx`.

https://nextjs.org/docs/getting-started/installation